src/cascadia/TerminalControl/XamlUiaTextRange.cpp

```diff
@@ -4,6 +4,11 @@
 #include "pch.h"
 #include "XamlUiaTextRange.h"
 #include "UiaTextRange.hpp"
+#include <UIAutomationClient.h>
+
+// the same as COR_E_NOTSUPPORTED
+// we don't want to import the CLR headers to get it
+#define XAML_E_NOT_SUPPORTED 0x80131515L

```

Comment:
wait this was a bad merge
this will not build

Reply:
wait what. is github having a laugh

Reply:
![image](https://user-images.githubusercontent.com/11050425/73576603-acb64700-442f-11ea-8b39-ff55d90a340e.png)

🤣 hahahahaha

Reply:
I'll fix it. Don't worry.